Sources/API/Alias.swift

```diff
@@ -0,0 +1,60 @@
+//
+//  Alias.swift
+//  SwiftPrettyPrint
+//
+//  Created by Yusuke Hosonuma on 2020/03/06.
+//
+
+// ----------------------------------------------------------------------------
+//
+// Provide a simplicity description for print.
+//
+// No need to enclose in parenthese unlike `print()` or `debugPrint()`.
+// This is convenient for print to written expression.
+//
+// For example:
+//
+// ```
+// let string = "Hello, world."
+// string.hasPrefix("Hello")
+//
+// // when use `print()` - needs enclose in parenthese
+// Debug.print(string.hasPrefix("Hello"))
+//
+// // when use `p` - not needs enclose in parenthese
+// Debug.p >> string.hasPrefix("Hello")
+// ```
+//
+// ----------------------------------------------------------------------------
+
+extension Debug {
+    public struct P {}
+    public struct PP {}
+    public struct PD {}
+    public struct PPD {}
+
+    public static var p: P { P() }
+    public static var pp: PP { PP() }
+    public static var pd: PD { PD() }
+    public static var ppd: PPD { PPD() }
+}
+
+@discardableResult
+public func >> (_: Debug.P, target: Any) -> String {
```

Comment:
I chosen `>>` because looks are intuitively like starlight for programmers, and less likely to conflict.